File: Find Unique Binary String.py
# https://leetcode.com/problems/find-unique-binary-string/?envType=daily-question&envId=2025-02-20
# Find Unique Binary String

class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        
        n = len(nums)
        answer = []

        for i in range(n):
            if nums[i][i] == '0':
                answer.append('1')
            else:
                answer.append('0')

        return "".join(answer)

File: Find Unique Binary String.js
// https://leetcode.com/problems/find-unique-binary-string/?envType=daily-question&envId=2026-03-08
// Find Unique Binary String

/**
 * @param {string[]} nums
 * @return {string}
 */
var findDifferentBinaryString = function(nums) {
    
        const n = nums.length;
        const answer = [];

        for (let i = 0; i < n; i++) {
            if (nums[i][i] === '0') {
                answer.push('1');
            } else {
                answer.push('0');
            }
        }

        return answer.join("");
};